internal/bot/bot.go

```diff
@@ -223,6 +228,19 @@ func (b *Bot) createAnnotations(ctx context.Context, pr *github.PullRequest, r *
 	var annotations = []*github.CheckRunAnnotation{}
 
 	for _, f := range parsedDiff.Files {
+		// Skip over any files listed in `ignore`
+		if ignorePatterns := ic.Ignore; ignorePatterns != nil {
```

Comment:
turn this into a function `if IgnoredByRepo(...) ... continue`?

Reply:
32038e2